ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -309,6 +314,18 @@ def test_function(
         target_fn, *args, test_compile=test_flags.test_compile, **kwargs
     )
 
+    if test_flags.with_copy:
+        array_fn = ivy.is_array
+        if "copy" in list(inspect.signature(target_fn).parameters.keys()):
```

Comment:
We shouldn't do this kind of search in arguments for each example, but instead this logic should be moved into the decorator where this is done only once.

Reply:
By decorator, do you mean the frontend `inputs_to_ivy_arrays` decorator?